Cssnavbartop

PierreDmyy opened the pull request:
NavbarTop with styles

PierreDmyy committed:
css on component NavBarTop

PierreDmyy committed:
ok
Merge branch 'css' of github.com:WildCodeSchool/reims-0218-gyvit into css

PierreDmyy committed:
search bar

PierreDmyy committed:
ok

Merge branch 'develop' of github.com:WildCodeSchool/reims-0218-gyvit into css

dadtmt commented on src/components/NavbarTop.js:
```diff
@@ -7,8 +7,20 @@ import NavbarTopNotif from "./NavbarTopNotif"
+  lastname: "Duflot",
+  firstname: "Romain",
+  business: "Notoriety"
+}
```
```
({profile, notif})
```

dadtmt reviewed (changes requested):
run npm test and fix stories

PierreDmyy committed:
ok

Merge branch 'develop' of github.com:WildCodeSchool/reims-0218-gyvit into cssnavbartop

PierreDmyy committed:
props profile and notif
